bgpd/bgp_evpn.c

```diff
@@ -2496,8 +2496,9 @@ static int install_evpn_route_entry_in_vrf(struct bgp *bgp_vrf,
 		pi = bgp_create_evpn_bgp_path_info(parent_pi, dest, &attr);
 		new_pi = true;
 	} else {
-		if (attrhash_cmp(pi->attr, &attr)
-		    && !CHECK_FLAG(pi->flags, BGP_PATH_REMOVED)) {
+		if (attrhash_cmp(pi->attr, &attr) &&
+		    attrhash_cmp(pi->extra, parent_pi->extra) &&
```

Comment:
Isn't attrhash_cmp just for struct attr? I don't understand or am I missing something here... :) And pi->extra can be NULL btw.

Reply:
done